docs/conf.py

```
@@ -0,0 +1,185 @@
+myst_heading_anchors = 3
+
+# -- Options for {MyST}NB ----------------------------------------------------
+
+nb_execution_mode = "cache"
+nb_execution_raise_on_error = True
+
+
+class CDAStyle(UnsrtStyle):
+    """Custom style for including PDF links."""
+
+    def format_url(self, _e: Entry) -> HRef:  # noqa: PLR6301
```

Ruff (0.14.8)

Unused `noqa` directive (non-enabled: `PLR6301`)

Remove unused `noqa` directive

(RUF100)